static/src/js/app/vuex/tests/mutations.spec.js

```diff
@@ -23,21 +21,49 @@ describe('Mutations', () => {
   });
 
   describe('createPersonalVocab', () => {
-    it('successfully updates the state', () => {
-      const state = {
-        personalVocabAdded: false,
-      };
-      const data = { created: true };
+    const vocabData = {
+      id: 19,
+      headword: 'foo',
+      gloss: 'bar',
+      familiarity: 1,
+      node: null,
+    };
+    const data = {
+      created: true,
+      data: vocabData,
+    };
+
+    it('successfully updates the state without entries fetched', () => {
```

Comment:
I don't really understand what this is testing -- the opposite of the one below? Why?

Reply:
Tests what would happen if you use the quick add vocab feature if your personal vocab list data has not yet been fetched. For instance, a user could use the quick add vocab button, but not have visited the personal vocab page, which would mean that the users personal vocab entries have not been fetched, and therefor `.entries` should still be `undefined`.

The one below is the same test, but where the entries have been fetched, and it tests that `.entries` is mutated.